sale_delivery_rate/demo/delivery_carrier_rate.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<!--
+    Copyright 2017 LasLabs Inc.
+    License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
+-->
+
+<odoo>
+    <record id="demo_car_rate_1" model="delivery.carrier.rate">
```

Comment:
Is this an OCA rule? I don't see anything about this in the style guide and kind of prefer the tighter spacing myself.